src/components/FilterTags/_filter-tags.scss

```diff
@@ -1,9 +1,50 @@
+@import "vanilla-framework/scss/settings_colors";
+
 .p-filter-tags {
   padding: 0.75rem;
+  position: relative;
 
   &__input {
     font-size: 0.875rem;
     margin: 0;
     padding: 0.25rem 0.5rem;
   }
 }
+
+.p-filter-panel {
+  margin-top: 0.5rem;
+  opacity: 0;
+  position: absolute;
+  top: 2.75rem;
+  transition: opacity 0.5s;
+  width: calc(100% - 1.5rem);
+
+  &.is-visible {
+    opacity: 1;
+  }
+
+  &__heading {
+    border-bottom: 1px solid $color-mid-x-light;
+    color: $color-mid-dark;
+    font-size: 0.75rem;
+    text-transform: uppercase;
+  }
+
+  &__list {
+    margin-bottom: 0.5rem;
+  }
+
+  &__item {
```

Comment:
This should probably have `cursor: pointer`.

And some hover style, but that can be a follow-up.